_data/projects/maven-test-bench.yml

```
@@ -8,7 +8,7 @@ tags:
 - performance
 - quickperf
 upforgrabs:
-  name: ":thumbsup: good first issue"
+  name: good first issue
```

@jeanbisutti Here, also you will have to revert.
```suggestion
  name: : "thumbsup: good first issue"
```